### PR TITLE
Misc Ameridian Changes

### PR DESCRIPTION
**Pull request by r4d6**
## About The Pull Request
Fixes and balances to ameridian stuff : 
\- Cut the research given my ameridian cores by half
\- Prevent literally everything from harvesting crystals
\- Make the blue crystals completely invisible when destroyed
\- Slowed down the growth of the red crystals
\- Doubled the LA cost of Ameridian Cores
\- Prevent ghosts from controlling dead tenders
\- Added a dead sprite for the ameridian tender
\- Change the crystals lighting to scale with their growth level.

### CODE REVIEW

**Comment by Yawet330** [comment]
Suggestion:  Change it from a 1.0 flat growth rate to a 2.0, a .5 reduction would still make red worth it, as unlike green, you cannot spam liquid tiberium sprayers to make insane cash.

Keep in mind, most edge-cases where reds outperformed Greens where when I used specific structures and viscerators to force the most efficient growth pattern possible.

A /slight/ nerf is needed, but a full 1.5 reduction is not imo

**Comment by r4d6** [comment]
> Suggestion: Change it from a 1.0 flat growth rate to a 2.0, a .5 reduction would still make red worth it, as unlike green, you cannot spam liquid tiberium sprayers to make insane cash.
> 
> Keep in mind, most edge-cases where reds outperformed Greens where when I used specific structures and viscerators to force the most efficient growth pattern possible.
> 
> A /slight/ nerf is needed, but a full 1.5 reduction is not imo

First, it is a probability-based chance. So if it is set to 2, then it will have a 2% chance, every time the crystal tick, to grow/spread.

Second, there's an efficient growth pattern?

File: code/modules/ameridian/ameridian_crystal_blue.dm [hunk 1]
```diff
@@ -77,6 +77,8 @@
 /obj/structure/ameridian_crystal/blue/harvest_crystals()
 	visible_message("[src] crumbles into a pile of crystals...")
 	icon_state = ""
+	for(var/U in underlays)
```
Comment (lolman360) [inline code/modules/ameridian/ameridian_crystal_blue.dm:80]:
```suggestion
	underlays.Cut()
```

Comment (r4d6) [reply code/modules/ameridian/ameridian_crystal_blue.dm]: Done

**Pull request merged**